fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/flight_model.cfg

```diff
@@ -399,44 +415,50 @@ Pump.18 = Name:RightInnerTankPumpAft#Pressure:20#DestinationLine:RightInnerTankP
 Pump.19 = Name:TrimTankPumpLeft#Pressure:20#DestinationLine:TrimTankPumpLeftToTrimLineJunction1#TankFuelRequired:Trim#Type:Electric#Index:19
 Pump.20 = Name:TrimTankPumpRight#Pressure:20#DestinationLine:TrimTankPumpRightToTrimLineJunction1#TankFuelRequired:Trim#Type:Electric#Index:20
 Pump.21 = Name:APUFeedPump#Pressure:5#DestinationLine:APUFeedPumpToAPUIsoValve#Type:APUDriven
+
 Trigger.1 = Name:InnerandMidTanksXferFeed1Start#Target:Feed1#Threshold:6436#Condition:TankQuantityBelow#EffectTrue:OpenValve.FeedTank1FwdTransferValve1,OpenValve.FeedTank1FwdTransferValve2,StartPump.LeftInnerTankPumpFwd,StartPump.RightInnerTankPumpFwd,SetJunction.FwdGalleryJunction.2
 Trigger.2 = Name:InnerandMidTanksXferFeed2Start#Target:Feed2#Threshold:6857#Condition:TankQuantityBelow#EffectTrue:OpenValve.FeedTank2FwdTransferValve1_1,OpenValve.FeedTank2FwdTransferValve2_1,StartPump.LeftInnerTankPumpFwd,StartPump.RightInnerTankPumpFwd,SetJunction.FwdGalleryJunction.2
 Trigger.3 = Name:InnerandMidTanksXferFeed3Start#Target:Feed3#Threshold:6857#Condition:TankQuantityBelow#EffectTrue:OpenValve.FeedTank3FwdTransferValve1_1,OpenValve.FeedTank3FwdTransferValve2_1,StartPump.LeftInnerTankPumpFwd,StartPump.RightInnerTankPumpFwd,SetJunction.FwdGalleryJunction.2
 Trigger.4 = Name:InnerandMidTanksXferFeed4Start#Target:Feed4#Threshold:6436#Condition:TankQuantityBelow#EffectTrue:OpenValve.FeedTank4FwdTransferValve1,OpenValve.FeedTank4FwdTransferValve2,StartPump.LeftInnerTankPumpFwd,StartPump.RightInnerTankPumpFwd,SetJunction.FwdGalleryJunction.2
 Trigger.5 = Name:Equalize1and4#Target:Feed1,Feed4#Threshold:2#Condition:TankAbsImbalanceBelow#EffectTrue:OpenValve.FeedTank1FwdTransferValve2,OpenValve.FeedTank4FwdTransferValve2,OpenValve.FeedTank1AftTransferValve1,OpenValve.FeedTank4AftTransferValve1
 Trigger.6 = Name:Equalize2and3#Target:Feed2,Feed3#Threshold:2#Condition:TankAbsImbalanceBelow#EffectTrue:OpenValve.FeedTank2FwdTransferValve2_1,OpenValve.FeedTank3FwdTransferValve2_1,OpenValve.FeedTank2FwdTransferValve2_2,OpenValve.FeedTank3FwdTransferValve2_2,OpenValve.FeedTank2AftTransferValve1,OpenValve.FeedTank3AftTransferValve1
-Trigger.7 = Name:Innerand MidTanksXferFeed1and4End#Target:Feed1#Threshold:6765#Condition:TankQuantityAbove#EffectTrue:CloseValve.FeedTank1FwdTransferValve1,CloseValve.FeedTank1FwdTransferValve2,CloseValve.FeedTank4FwdTransferValve1,CloseValve.FeedTank4FwdTransferValve2,SetJunction.FwdGalleryJunction.1,SetJunction.FeedTanks2_3Junction1.1,SetJunction.FeedTanks2_3Junction1.1
+Trigger.7 = Name:Innerand MidTanksXferFeed1and4End#Target:Feed1#Threshold:6765#Condition:TankQuantityAbove#EffectTrue:CloseValve.FeedTank1FwdTransferValve1,CloseValve.FeedTank1FwdTransferValve2,CloseValve.FeedTank4FwdTransferValve1,CloseValve.FeedTank4FwdTransferValve2,SetJunction.FwdGalleryJunction.1  ;
```

Comment:
no comment after comment symbol so probably better to remove it

Reply:
Thanks, fixed